riotfile.py

```diff
@@ -1120,6 +1120,45 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             },
             command="pytest {cmdargs} tests/contrib/aiopg",
         ),
+        Venv(
+            name="aiohttp",
+            command="pytest {cmdargs} tests/contrib/aiohttp",
+            pkgs={"pytest": [">=3"], "pytest-aiohttp": [latest]},
+            venvs=[
+                Venv(
+                    pys=["3.5", "3.6"],
+                    pkgs={
+                        "aiohttp": ["~=2.0", "~=2.1", "~=2.2"],
+                        "aiohttp_jinja": ["~=0.1.2", "~=0.1.3"],
+                        "yarl": "~=0.18.0",
+                    },
+                ),
+                Venv(
+                    pys=["3.5", "3.6", "3.7", "3.8"],
+                    pkgs={
+                        "aiohttp": "~=2.3",
+                        "aiohttp_jinja": "~=0.1.5",
+                        "yarl": "~=1.0",
+                    },
+                ),
+                Venv(
+                    pys=["3.5", "3.6", "3.7"],
```

Comment:
```suggestion
                    pys=select_pys(min_version="3.5", max_version="3.7"),
```

can do the same with the others as well